src/Qossmic/ExceptionHandling/ExceptionHandlerRegistry.php

```diff
@@ -45,6 +45,7 @@ public function get(string $strategy): ExceptionHandlerInterface
             throw new \InvalidArgumentException(sprintf('The exception handling strategy "%s" is not registered (use one of ["%s"]).', $strategy, implode(', ', array_keys($this->strategies))));
         }
 
+        /* @phpstan-ignore-next-line */
```

Comment:
I am not sure, but I think with the `phpstan/phpstan-symfony` extension phpstan might be able to infer the type here

Reply:
I'm gonna check for that in a follow-up PR